README.md

```diff
@@ -1,2 +1,167 @@
-# IDR_stream
-Software for feature extraction from IDR image data
+# idrstream
+
+Software for feature extraction from IDR image data!
+
+## About
+
+Idrstream is a package for deriving features associated with metadata from medical image data hosted on [IDR](https://idr.openmicroscopy.org/).
+
+It uses the following tools to process data:
+
+- [Aspera](https://www.ibm.com/products/aspera) high-speed transfer client is used to download IDR data.
+More details can be found on the [IDR website](https://idr.openmicroscopy.org/about/download.html) and in setup/usage sections.
+- [Fiji](https://imagej.net/software/fiji/) is used with [PyImagej](https://github.com/imagej/pyimagej) to read mitosis movies, which are hosted on IDR in [CellH5](https://github.com/CellH5/cellh5) format.
+- [BaSiCPy](https://github.com/peng-lab/BaSiCPy) is used to perform illumination correction on images.
+BaSiCPy is the python implementation of the BaSiC method for illumination correction, introduced in [Peng, T et al., 2020](https://doi.org/10.1007/978-3-030-59722-1_17).
+- [CellPose](https://github.com/mouseland/cellpose) is used to segment nuclei from mitosis movies.
+CellPose was first introduced in [Stringer, C., Wang, T., Michaelos, M. et al., 2020](https://doi.org/10.1038/s41592-020-01018-x).
+- [DeepProfiler](https://github.com/cytomining/DeepProfiler) is used to extract features from mitosis movies. 
+- [PyCytominer](https://github.com/cytomining/pycytominer) is used to compile DeepProfiler-extracted features with their metadata. 
```

Comment:
I thought that we decided to stop IDR stream at the single-cell DP feature extraction step. (i.e. no need for pycytominer)

Reply:
I think we discussed not normalizing features with PyCytominer, but PyCytominer is still very useful for compiling single-cell feature data into one pandas dataframe. The DP feature output is much more difficult to work with than the PyCytominer `get_single_cells()` output of single cell features and their metadata.